modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/code_mirror_keyboard_message/CodeMirrorKeyboardMessage.tsx

```diff
@@ -19,7 +19,15 @@ import React from 'react';
 
 import './CodeMirrorKeyboardMessage.scss';
 
-export default function CodeMirrorKeyboardMessage({className, keyIsEnabled}) {
+interface IProps {
+	className?: string;
+	keyIsEnabled: boolean;
+}
```

Comment:
We don't need `propTypes` if we have this.

Reply:
I forgot to remove it, thanks for noticing this